.prettierrc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
There's already prettier in `@redhat-cloud-services/eslint-config-redhat-cloud-services` any reason to use smaller width, `jsxSingleQuote` and turning off `bracketSpacing`?

Reply:
I copied our `.prettierrc` from OpenShift Console to solve the `trailingComma` issue... and then just left what we had. No real reason for declaring anything else. I failed to extend the existing ones more so because I misunderstood the way prettier worked and didn't see a problem with the output. Although, 150 cols does seem a bit wide, but I have extended and only added the trailing comma now.